__tests__/app.test.js

```diff
@@ -312,12 +312,27 @@ describe("/api", () => {
       return request(app)
         .patch("/api/articles/1000")
         .send({
-          inc_votes: 5
+          inc_votes: 5,
         })
         .expect(404)
         .then((response) => {
           expect(response.body.msg).toBe("Article not found");
         });
     });
   });
+  describe("DELETE /api/comments/:comment_id", () => {
+    test("DELETE 204: Should delete given comment by comment_id", () => {
+      return request(app).delete("/api/comments/1").send().expect(204);
+    });
```

Comment:
Good test, just a tip you don't need `.send()` at all here. Expecting 204 back is correct.